system/BaseModel.php

```diff
@@ -421,16 +432,16 @@ abstract protected function doUpdate($id = null, $data = null): bool;
      * Compiles an update and runs the query.
      * This method works only with dbCalls.
      *
-     * @param array|null  $set       An associative array of update values
-     * @param string|null $index     The where key
-     * @param int         $batchSize The size of the batch to run
-     * @param bool        $returnSQL True means SQL is returned, false will execute the query
+     * @param array|null               $set         An associative array of update values
+     * @param array|RawSql|string|null $constraints
```

Comment:
`RawSql` seems to be SQL database specific.
Any kind of database should use it?

Reply:
Its a Builder thing anyways. I probably should move this logic to Model.